uint8 valid
float32 temperature
float32 humidity
